### PR TITLE
[libvips] build against lcms/libspng, fix AVIF/JXL fuzzer

### PR DESCRIPTION
**Pull request by kleisauke**
See the individual commits for more details.

/cc @jcupitt

### CODE REVIEW

**Review by DavidKorczynski — APPROVED** [review]
This looks good to me. Will wait for approval by @jcupitt before merging.

**Comment by jcupitt** [comment]
Looks good here too. Thanks for taming this dragon, Kleis.

**Pull request merged**
